### ISSUE TITLE
About different training settings for ScanNet and S3DIS

### ISSUE THREAD

**Issue by haibo-qiu** [Gofinge/PointTransformerV2#19]
Hi @Gofinge,
Thanks for your excellent work and code👍  

I have noticed that there are several different settings for training on scannet and s3dis, for example, different depths for patch embedding, encoder, and decoder. From the paper, your experiments including comprehensive ablation studies focus on scannet, but have you tried the same architecture of scannet on s3dis?

Also, the unpooling backend is different between scannet and s3dis. The ablation study confirms that using map unpooling brings gains on scannet, but the interpolation unpooling is used on s3dis.
https://github.com/Gofinge/PointTransformerV2/blob/9b6faeb73c8e34e1b7e3fbb6a2f30dd980ac4bcb/configs/s3dis/semseg-ptv2m2-0-base.py#L33
https://github.com/Gofinge/PointTransformerV2/blob/9b6faeb73c8e34e1b7e3fbb6a2f30dd980ac4bcb/configs/scannet/semseg-ptv2m2-0-base.py#L34

Much appreciate it if you can give me some insights~

**Comment by Gofinge** [comment]
Hi, the S3DIS performance we reported in our paper was achieved by a similar architecture to the setting for ScanNet. While to release a better codebase, I re-tuning some of the parameters. Since the init voxel size of S3DIS is twice as the ScanNet, I reduced one encoding-decoding stage, I also try different unpooling methods (still with our grid-pooling method). Actually, there is no significant difference between different settings, since training on S3DIS is unstable.
